test/data/binned_contigs_empty/out_header.summary

```diff
@@ -1,4 +1,4 @@
 Bin name	Completeness	Genome size	GC content
 out_header.001.fasta	100.0%	2674902	52.9
-out_header.002.fasta	100.0%	3270493	32.7
-out_header.003.fasta	92.5%	2452188	54.8
+out_header.002.fasta	100.0%	5951	32.7
+out_header.003.fasta	100.0%	5846	54.8
```

Comment:
Removed large portions of the example files in the `binned_contigs_empty` directory to save on space, since they are there as input for a test that fails.